src/services/logsSyntaxMatches.ts

```diff
@@ -0,0 +1,18 @@
+// Synced with grafana/grafana/public/app/features/logs/components/panel/grammar.ts - July 3, 2025
```

Comment:
Pretty cool. You can also put a permalink to the current version from the "get permalink" button https://github.com/grafana/grafana/blob/ca730935733d86339177ed5d014b9343831df98b/public/app/features/logs/components/panel/grammar.ts